provider/datagen/src/transform/cldr/fallback/mod.rs

```diff
@@ -155,7 +155,7 @@ impl From<&cldr_serde::parent_locales::Resource> for LocaleFallbackParentsV1<'st
                 // We always fall back from language-script to und
                 continue;
             }
-            parents.insert(source.write_to_string().as_bytes(), &target.into());
+            parents.insert(source.write_to_string().as_bytes().into(), &target.into());
```

Comment:
`.as_str().into()` because we know we have a `str` and don't need to deal with bytes.

Reply:
There is no `.as_str()` on `Cow<str>` sadly. I would use `.borrow()` or `.deref()` but those require importing the trait. I used the `&*` operator which does the trick but requires an extra set of parentheses.